data/os/Debian.yaml

```diff
@@ -16,6 +16,7 @@ powerdns::authoritative_extra_packages:
 powerdns::authoritative_service_name: pdns
 powerdns::authoritative_configdir: /etc/powerdns
 powerdns::authoritative_config: "%{lookup('powerdns::authoritative_configdir')}/pdns.conf"
+powerdns::authoritative_group: 'pdns'
```

Comment:
isn't this always the default? shouldn't we move this to the einit.pp?

Reply:
I don't have an idea if this is also the default on e.g. FreeBSD and arch.

Reply:
Checked the module again, and I think it should be safe to set the group as default.

But I wonder: why in the init.pp and not in data/common.yaml? Do we have any rules for that?